gitlab-pipeline/stage/test.yml

```diff
@@ -262,6 +262,7 @@ test_accep_vexpress_qemu_uboot_uefi_grub:
 # the build script does not generate the Mender Artifact neither.
 # This job collects the .ubifs for debugging but not for release publishing
 test_accep_vexpress_qemu_flash:
+test:acceptance:vexpress_qemu_flash:
```

Comment:
Forgot to remove original name

Reply:
Thanks, I was bound to miss something here, been staring at these for 30 mins straight :eyes: